README.md

```diff
@@ -1,2 +1,57 @@
-# YouTube-s-video-player-page-
-First Hmtl  Project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name: YouTube-s-video-player-page
+
+> In this project we strived to create a youtube video player page clone that was responsive and aesthetically pleasing. We used HTML & CSS3(Flexbox) to achieve this.
+![screenshot](./graphics/Screenshot.png)
+
+A Youtube clone project
+
+## Built With
+
+- HTML & CSS3,
+- NO frameworks,
+- Vscode
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites: Code Editor 
+
+### Setup : $git clone https://github.com/jurgen1c/YouTube-s-video-player-page-/tree/master
```

Comment:
- [ ] **[OPTIONAL]** The way you can put commands in your readme and they don't look as links is putting it between the backquotes (`): 
```
$git clone https://github.com/jurgen1c/YouTube-s-video-player-page-/tree/master
```